test/domains.jl

```diff
@@ -1,21 +1,25 @@
 @testset "Domains" begin
   @testset "PointSet" begin
-    ps = PointSet([1. 0.; 0. 1.])
-    @test nelms(ps) == 2
-    @test coordinates(ps, 1) == [1., 0.]
-    @test coordinates(ps, 2) == [0., 1.]
```

Comment:
Can you please undo these changes and add a separate section for the new constructor? We just need a small paragraph constructing a point set with a list of tuples and doing basic checks.

Reply:
We will be duplicating code. I don't like to duplicate code.